src/main/java/people/employees/Developer.java

```diff
@@ -17,17 +17,13 @@
 
 @ToString
 public class Developer extends Employee {
-    private static final int BASE_SENIOR_SALARY = 12000;
-    private static final int BASE_MID_SALARY = 5000;
-    private static final int BASE_JUNIOR_SALARY = 3000;
-
     private final Seniority seniority;
 
     @Getter
     private final LinkedList<TechStack> skills;
 
     public Developer() {
-        super(Position.DEVELOPER);
+        super("xd", "xd", Position.DEVELOPER);
```

Comment:
why xd, use generator, maybe put template names in json and get random name from this json and put to consturctor

Reply:
Changed xd to normal value